lib/ansible/modules/cloud/vmware/vmware_dvswitch.py

```diff
@@ -19,7 +19,7 @@
 short_description: Create or remove a distributed vSwitch
 description:
     - Create or remove a distributed vSwitch
-version_added: 2.0
+version_added: "2.5"
```

Comment:
Let this be as '2.0',

This value denotes on which version vmware_dvswitch was added.